src/app/academic-bg-form/page.jsx

```diff
@@ -53,6 +54,7 @@ export default function AcademicBgForm() {
         method: isEdit ? "PUT" : "POST"
       });
       toast.success("Suas informações acadêmica foram registradas com sucesso");
+      redirect(Routes.DASHBOARD);
```

Comment:
Esse redirecionamento está quebrando o envio do formulário, está acontecendo dois erros:
- Routes não foi importado para o arquivo
![image](https://github.com/user-attachments/assets/42c4006a-213a-43d6-9e03-0dfe818df1a7)
para corrigir basta adicionar nos imports 
```js
import { Routes } from "@/constants/routes";
```
- Esse redirecionamento não funciona em componentes client "use client"
![image](https://github.com/user-attachments/assets/229f9781-077b-41d6-8cd8-95abdb90b27f)

para corrigir basta seguir a abordagem do userRouter hook da [documentação](https://nextjs.org/docs/app/building-your-application/routing/redirecting#userouter-hook)
```js
import { useRouter } from 'next/navigation'

router.push(Routes.DASHBOARD);
```
**Atenção na importação do useRouter, existem 2 e se não for importado corretamentenão funcionará**